tfhe/src/shortint/ciphertext/compressed_ciphertext_list.rs

```diff
@@ -0,0 +1,48 @@
+use self::compressed_modulus_switched_glwe_ciphertext::CompressedModulusSwitchedGlweCiphertext;
+use crate::conformance::ParameterSetConformant;
+use crate::core_crypto::prelude::*;
+use crate::shortint::parameters::CompressedCiphertextConformanceParams;
+use crate::shortint::{CarryModulus, MessageModulus};
+
+#[derive(Clone, serde::Serialize, serde::Deserialize)]
+pub struct CompressedCiphertextList {
+    pub modulus_switched_glwe_ciphertext_list: Vec<CompressedModulusSwitchedGlweCiphertext<u64>>,
+    pub ciphertext_modulus: CiphertextModulus<u64>,
+    pub message_modulus: MessageModulus,
+    pub carry_modulus: CarryModulus,
+    pub pbs_order: PBSOrder,
+    pub lwe_per_glwe: LweCiphertextCount,
+    pub count: usize,
+}
+
+impl ParameterSetConformant for CompressedCiphertextList {
+    type ParameterSet = CompressedCiphertextConformanceParams;
+
+    fn is_conformant(&self, params: &CompressedCiphertextConformanceParams) -> bool {
+        let len = self.modulus_switched_glwe_ciphertext_list.len();
+
+        if len == 0 {
+            return false;
+        }
+
+        let count_is_ok = self.modulus_switched_glwe_ciphertext_list[..len - 1]
+            .iter()
+            .all(|a| a.bodies_count() == params.lwe_per_glwe);
+
```

Comment:
missing last one

Reply:
Removed when removed `total_count`
Error is returned in `get` if the total count is too small

Reply:
the last one should have a check that bodies_count <= params.lwe_per_glwe